seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -45,23 +45,25 @@
       "id": "587d8250367417b2b2512c5e",
       "title": "Learn how a Stack Works",
       "description": [
-        "You are probably familiar with stack of books on your table. You might have used the undo feature in your computer. You are also used to hitting the back button on your phone to go back to the previous view in your app.",
+        "You are probably familiar with stack of books on your table. You might have used the undo feature in your computer. You are also probably used to hitting the back button on your phone to go back to the previous view in your app.",
```

Comment:
I recommend changing "You might have used the undo feature in your computer." to "You have likely used the Undo feature of a text editor".

"In your computer" seems generic.